lib/generators/shopify_app/install/templates/shopify_session_repository.rb

```diff
@@ -1,23 +0,0 @@
-# You should replace InMemorySessionStore with what you will be using
```

Comment:
Only thing we lose is this documentation now. Is there any other place we could put it?